stable/aws-vpc-cni/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: aws-vpc-cni
-version: 1.18.1
+version: 1.3.0
```

Comment:
Why is this 1.3.0? https://github.com/aws/amazon-vpc-cni-k8s/blob/master/charts/aws-vpc-cni/Chart.yaml#L3 seems like we are matching the release version.

Reply:
Right, but it would fail if I keep it 1.18.1.. We have kept it different in the past 
```
eks/aws-vpc-cni	1.13.0       	v1.13.0    	A Helm chart for the AWS VPC CNI
eks/aws-vpc-cni	1.2.8        	v1.12.6    	A Helm chart for the AWS VPC CNI
eks/aws-vpc-cni	1.2.7        	v1.12.5    	A Helm chart for the AWS VPC CNI
eks/aws-vpc-cni	1.2.6        	v1.12.2    	A Helm chart for the AWS VPC CNI
eks/aws-vpc-cni	1.2.2        	v1.12.1    	A Helm chart for the AWS VPC CNI
```

Reply:
yeah it looks odd, sure we can have it 1.3.0 for now given that we will have another release in few weeks